### ISSUE TITLE
About the effectiveness

### ISSUE THREAD

**Issue by SunSiShining** [xjtushujun/meta-weight-net#6]
Thank you for your excellent work!

Here, we raised some questions about the effectiveness (time & memory) of Meta-Weight-Net：

Compared to Learn-to-Reweight (Ren, 2017), how about the cost of running time and GPU memory per training step? 

Can the training process's time efficiency be improved by updating Meta-Weight-Net every several steps (rather than updating every step)? Will this affect model's performance? 

Is it possible to achieve multi-GPU parallelism (based on Pytorch)?

Thanks very much~

**Comment by bigheiniu** [comment]
For multiple steps, you can check the idea in this paper. 
Meta Label Correction for Noisy Label Learning

**Comment by Sean-asc** [comment]
Hi, I am also curious about  the cost of running time, So, what's the verdict？
